### DISCUSSION TITLE
How can I change the theme and the type of the app launcher?

### DISCUSSION THREAD

**Discussion by Danidro20** [JaKooLit/Hyprland-Dots#375] [Q&A]
Sorry if this is somewhere in the guide, I did not find it.
I want to change the theme and the type of the app launcher like this screenshot or like the showcase video, but I don't know how to do it.
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/70298813/7bbc9e73-23e7-4782-b83a-25b82f901fe9)

**Comment by JaKooLit** [comment]
thats wofi

and its the launcher that I used before

sorry but I dont provide such kind of guide

**Reply by Danidro20** [reply]
Aahh ok ok, don't worry.
Thank you for the dots and the script!
